.expeditor/end_to_end.pipeline.yml

```diff
@@ -18,6 +18,18 @@ steps:
 # E2E
 #######################################################################
 
+  - label: ":docker: Docker End-to-End Supervisor Tests"
+    command:
+      - cd automated_testing
+      # TODO (CM): should be DEV, but somehow it doesn't have the packages I need at the moment?!
+      - ./run_all.sh stable
```

Comment:
Testing this now, but my suspicion based on the output of `run_all.sh DEV` is that the `Dockerfile`s need an `ARG` to pass in `HAB_BLDR_URL`.

Reply:
Looks like that was it.   I hard-coded `CHANNEL` and `HAB_BLDR_URL` in the `Makefile` to get this to work.  Do you think worth the effort of making `HAB_BLDR_URL` a parameter though at this point? If so, I can work up a patch for you. 

```Step 9/9 : RUN hab pkg install core/hab --binlink --force --channel="${CHANNEL}"
 ---> Running in 5988014cf364
» Installing core/hab
☁ Determining latest version of core/hab in the 'DEV' channel
↓ Downloading core/hab/0.88.0/20191009204957
☛ Verifying core/hab/0.88.0/20191009204957
✓ Installed core/hab/0.88.0/20191009204957
★ Install of core/hab/0.88.0/20191009204957 complete with 1 new packages installed.
» Binlinking hab from core/hab/0.88.0/20191009204957 into /bin
★ Binlinked hab from core/hab/0.88.0/20191009204957 to /bin/hab```

Reply:
I think it's OK now that we're in production Builder.